### ISSUE TITLE
[Firefox 89+][Proton] Disable 'Proton' settings to keep compatibility with CustomCSSforFx 

### ISSUE THREAD

**Issue by Aris-t2** [Aris-t2/CustomCSSforFx#339]
**The versions 3.2.0-3.2.2+ of this projects files already made many features compatible to Proton.**

--------------------------------

New **Firefox 89** release date is _June 1st 2021_. CSS code offered by CustomCSSforFx is not fully compatible to Proton.

Open `about:config` and disable Proton settings:

`browser.proton.enabled > false` (disables Proton UI of toolbars, tabs, application menu)
`browser.proton.contextmenus.enabled > false` (disables Proton UI of context menus)
`browser.proton.doorhangers.enabled > false` (disables Proton UI of doorhangers)

Other Proton settings:
`browser.proton.infobars.enabled > false`
`browser.proton.modals.enabled > false`
`browser.proton.places-tooltip.enabled > false`
`browser.proton.toolbar.version > 0-3` (no effect atm)

-------------

Use `tabs_below_navigation_toolbar_fx65_v2` settings as a workaround for `tabs below navigation bar`:

```css
/* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_macOS.css"; /**/
```

**Comment by Pizzapops** [comment]
**Mozilla Proton Development Team** might also be working  on the speculated Mozilla keyboard.
![Dilbert_200111](https://user-images.githubusercontent.com/33039961/114619503-3f778600-9c70-11eb-97d2-8bdab0710e2b.gif)

**Comment by nicolaasjan** [comment]
Since today on Nightly 90.0a1(2021-04-21), this workaround doesn't work properly any more (it did before).

I use the setting "tabs below navigation toolbar", but now the tabs are at the very bottom of the window...
Also, just the active tab is visible. The rest is hidden.
Screenshot:
![screenshot-nightly](https://user-images.githubusercontent.com/14093220/115686517-8ee94080-a359-11eb-9f6d-caad1d67a024.png)

**Comment by Aris-t2** [comment]
You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.

```css
/* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
```

**Comment by nicolaasjan** [comment]
> You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.
> 
> ```css
> /* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
> ```

Thanks!
That worked.

![screenshot-nightly2](https://user-images.githubusercontent.com/14093220/115827577-c664f500-a40c-11eb-9990-3c0a47f37abe.png)

**Comment by Speravir** [comment]
[Sören](https://www.soeren-hentzschel.at/), usually well informed about Firefox, says in final Fx 89 proton cannot be disabled anymore. :-(
(Cf. these postings of him in Camp-Firefox.de: [#1](https://www.camp-firefox.de/forum/thema/132325/?postID=1168270#post1168270) and [#2](https://www.camp-firefox.de/forum/thema/132418/?postID=1169639#post1169639), written in German.)

**Comment by glennpc** [comment]
If Firefox won't let me disable Proton any longer, I will be forced to stay with version 88.

I really don't like the looks and changes of what Proton has done to Firefox. This is a really bad move by Mozilla!

**Comment by nicolaasjan** [comment]
> If Firefox won't let me disable Proton any longer, I will be forced to stay with version 88.
> 
> I really don't like the looks and changes of what Proton has done to Firefox. This is a really bad move by Mozilla!

I hope @Aris-t2 will be able to modify the css files/create new ones when Proton can no longer be disabled.
And that the result will be similar to what can be seen in my [screenshot](https://user-images.githubusercontent.com/14093220/115827577-c664f500-a40c-11eb-9990-3c0a47f37abe.png) above.

Of course it will be a huge undertaking...

**Comment by MrTatiTunes** [comment]
> 
> 
> You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.
> 
> ```css
> /* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
> ```

How to use this code? is it line in userChrome or what? I had an identical situation with tabs out of the window at the bottom.

**Comment by MrTatiTunes** [comment]
> 
> 
> > You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.
> > ```css
> > /* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
> > /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
> > /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
> > ```
> 
> Thanks!
> That worked.
> 
> ![screenshot-nightly2](https://user-images.githubusercontent.com/14093220/115827577-c664f500-a40c-11eb-9990-3c0a47f37abe.png)

How you use this code... please expalin.

**Comment by nicolaasjan** [comment]
First of all read [the instructions](https://github.com/Aris-t2/CustomCSSforFx#instructions--howto--readme) carefully.

To edit the `userChrome.css` file, I recommend to use an advanced text editor (on Windows e.g. [Notepad++](https://notepad-plus-plus.org/)).

In the `userChrome.css` file, find the line (line 353 at the moment):
`/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/`

Uncomment (remove `/*`) this line.
So it becomes:
`@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/`

Now it should work.
As you can see, I have the menu bar at the top; without it, this solution has glitches.

**Comment by MrTatiTunes** [comment]
> 
> 
> First of all read [the instructions](https://github.com/Aris-t2/CustomCSSforFx#instructions--howto--readme) carefully.
> 
> To edit the `userChrome.css` file, I recommend to use an advanced text editor (on Windows e.g. [Notepad++](https://notepad-plus-plus.org/)).
> 
> In the `userChrome.css` file, find the line (line 353 at the moment):
> `/* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/`
> 
> Uncomment this line (remove `/*`).
> So it becomes:
> `@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/`
> 
> Now it should work (you may have to restart Firefox).
> As you can see, I have the menu bar at the top; without it, this solution has glitches.

OK but where I will find all userChrome.css
to edit 353-rd line with this instruction?

**Comment by nicolaasjan** [comment]
> OK but where I will find all userChrome.css
> to edit 353-rd line with this instruction?

Download the zip file, which contains the necessary files and folders.
Go to:
https://github.com/Aris-t2/CustomCSSforFx
and click on the green "Code" button at the top right of the page and click on "Download ZIP".
Extract it somewhere and copy the **entire** content of the `./CustomCSSforFx-master/classic/` subfolder to the `chrome` folder in your Firefox profile.
If this folder doesn't exist there yet, create it.

These instructions are also [here](https://github.com/Aris-t2/CustomCSSforFx#where-to-find-firefox-profile-folder-the-correct-location-for-user-styles).

Then you can edit the `userChrome.css` file.

Don't forget to "[Unlock custom CSS usage in Firefox 69 and newer](https://github.com/Aris-t2/CustomCSSforFx#unlock-custom-css-usage-in-firefox-69-and-newer)":
`about:config` > `toolkit.legacyUserProfileCustomizations.stylesheets` > `true`

As I already said, you should have read [the instructions](https://github.com/Aris-t2/CustomCSSforFx#instructions--howto--readme) carefully. ;)

**Comment by Aris-t2** [comment]
@Speravir

That would be really bad. Even if Firefox 89 final would allow disabling Proton, the switches for is would be gone at some point in the future anyways. 

@MrTatiTunes 

Everything on how to use this projects files is described on the main page.

**Comment by MrTatiTunes** [comment]
> 
> 
> @Speravir
> 
> That would be really bad. Even if Firefox 89 final would allow disabling Proton, the switches for is would be gone at some point in the future anyways.
> 
> @MrTatiTunes
> 
> Everything on how to use this projects files is described on the main page.

please provide me link... I am a bit lost in the 1000s issues and lost in multilayer design of GitHub also…

**Comment by nicolaasjan** [comment]
> please provide me link... I am a bit lost in the 1000s issues and lost in multilayer design of GitHub also…

https://github.com/Aris-t2/CustomCSSforFx

**Comment by MrTatiTunes** [comment]
> 
> 
> @Speravir
> 
> That would be really bad. Even if Firefox 89 final would allow disabling Proton, the switches for is would be gone at some point in the future anyways.
> 
> @MrTatiTunes
> 
> Everything on how to use this projects files is described on the main page.

please provide me link... I am a bit lost in the 1000s issues and lost in multilayer design of GitHub also…

> 
> 
> > please provide me link... I am a bit lost in the 1000s issues and lost in multilayer design of GitHub also…
> 
> https://github.com/Aris-t2/CustomCSSforFx

OK I did download it and changed name to proper userConfig.css 
toolkit.legacyUserProfileCustomizations.stylesheets  is set to true
edited 353 line acording to your instruction and Firefox Nightly did not change anything in the layout...

@import "./css/tabs/tabs_below_navigation_toolbar_fx74_macOS_default_tabs.css"; /**/ 
this one...
In my logic
this import should come from kind of local folder
do I need to place something in profile with something what was called out in the line "tabs_below_navigation_toolbar_fx74_macOS_default_tabs.css"

How to rearrange the profile to get the result from the post of yourself...
(BTW your layout looks perfect where from can I import it to Nightly?)

> 
> 
> > > You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.
> > > ```css
> > > /* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
> > > /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
> > > /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
> > > ```
> > 
> > 
> > Thanks!
> > That worked.
> > ![screenshot-nightly2](https://user-images.githubusercontent.com/14093220/115827577-c664f500-a40c-11eb-9990-3c0a47f37abe.png)
>

**Comment by MrTatiTunes** [comment]
![obraz](https://user-images.githubusercontent.com/30302908/116381648-019c6500-a80d-11eb-878b-5e8cb3455d2d.png)
my FF Nightly still looks like this

**Comment by MrTatiTunes** [comment]
![obraz](https://user-images.githubusercontent.com/30302908/116381840-2c86b900-a80d-11eb-9ab0-5b8abc8447cc.png)

**Comment by MrTatiTunes** [comment]
nightly revision:
![obraz](https://user-images.githubusercontent.com/30302908/116383405-b7b47e80-a80e-11eb-8fc6-13e378ae4b57.png)

**Comment by MrTatiTunes** [comment]
when i put all folders proper place instructed from 
https://github.com/Aris-t2/CustomCSSforFx
my tabs again landed invisible at the bottom of the ff window.
;(

**Comment by MrTatiTunes** [comment]
With absolurely no way to manage or swith them from GUI

**Comment by MrTatiTunes** [comment]
![obraz](https://user-images.githubusercontent.com/30302908/116384266-9902b780-a80f-11eb-9af0-e7bafda40997.png)
![obraz](https://user-images.githubusercontent.com/30302908/116384319-a6b83d00-a80f-11eb-8b17-ab24a46d4542.png)
this how it looks like

**Comment by nicolaasjan** [comment]
@MrTatiTunes;
Have you applied the `about:config` changes that are required?
See the [first post](https://github.com/Aris-t2/CustomCSSforFx/issues/339#issue-855108522) at the top of this page.

**Comment by MrTatiTunes** [comment]
OK i will check again,

**Comment by MrTatiTunes** [comment]
Miraculously it went back to default true values
today was again an update of nightly.
Checking results again

**Comment by MrTatiTunes** [comment]
![obraz](https://user-images.githubusercontent.com/30302908/116386658-e6802400-a811-11eb-92e0-ebc6d5fb3085.png)
this is what I have about inport...64

**Comment by MrTatiTunes** [comment]
WOW!!!! finally:
![obraz](https://user-images.githubusercontent.com/30302908/116387266-8ccc2980-a812-11eb-817f-5cd67bae2ff3.png)
Thank you a million I owe you a beer whenever be in Nederland... make me contact you on priv!

**Comment by MrTatiTunes** [comment]
![obraz](https://user-images.githubusercontent.com/30302908/116467578-bebbab00-a867-11eb-840d-bcaaa1aece6e.png)
I think for perfect there should be arrow left and arrow right for moving to desired tabs, but I love this look and motive...

**Comment by MrTatiTunes** [comment]
Oh! nooooo... it is partial "X" so motive has not showing full tabs but show them also partially.

**Comment by nicolaasjan** [comment]
@MrTatiTunes:
 I have the arrows here...:

![screenshot-tabs](https://user-images.githubusercontent.com/14093220/116513130-6a491780-a8c9-11eb-8d0c-ab912a5238c4.png)

**Comment by MrTatiTunes** [comment]
> 
> 
> @MrTatiTunes:
> I have the arrows here...:
> 
> ![screenshot-tabs2](https://user-images.githubusercontent.com/14093220/116514946-f8be9880-a8cb-11eb-9532-9a7f3964a8ca.png)

Wow? You on Nightly?

I guess, if I  may ask, pack me all your Chrome folder and send me it zipped we will see if it works identically. How you set the black motif? my is Light.

**Comment by nicolaasjan** [comment]
> I guess, if I may ask, pack me all your Chrome folder and send me it zipped we will see if it works identically.

This is for Nightly (I think you only need `userChrome.css`):
[userChrome.css.zip](https://github.com/Aris-t2/CustomCSSforFx/files/6400767/userChrome.css.zip)

I can't tell if it will work exactly the same for you though, because I'm on a Linux system.

> How you set the black motif? my is Light.

I just changed the theme to Dark.
Menu Extra: ---> Addons and Themes ---> Dark

And I have set a new integer value in`about:config` called: `ui.systemUsesDarkTheme` with the value`1`
Then internal Firefox pages will also be dark themed.
Don't know if that works on Windows.

>  You have as well a bit different spacing among ribbons.

That's maybe because I have set `browser.compactmode.show` to true in `about:config`, so the option becomes available in the "Customize" window:

![screenshot-compact](https://user-images.githubusercontent.com/14093220/116603171-eb85c600-a92c-11eb-80af-425f8175e041.png)

**Comment by MrTatiTunes** [comment]
I can not get arrow left <> right on tabs, and still tabs are possible to be shown partially.

**Comment by MrTatiTunes** [comment]
> 
> 
> You could try `tabs_below_navigation_toolbar_fx65_v2.css` as a workaround, but it has its flaws.
> 
> ```css
> /* - Fx 65+ v2 - mainly for multi-lined tabs (Glitches with inactive menubar!) ******************/  /* <- use this for multiple tab lines support */
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
> /* @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2_force_hidden_menubar.css"; /**/ /* no menubar when OS titlebar is active */
> ```
I  can not get arrow left <> right on tabs, and still tabs are possible to be shown partially.
![obraz](https://user-images.githubusercontent.com/30302908/117261278-e3a3b580-ae47-11eb-9e7a-9f6621ed63da.png)

**Comment by MrTatiTunes** [comment]
> 
> 
> > I guess, if I may ask, pack me all your Chrome folder and send me it zipped we will see if it works identically.
> 
hmm! have you any idea where in this motif are arrows?

**Comment by hopalongrock** [comment]
@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
works fine in Fx89.0b
However in Customize I can't drag from/to the Navigation Toolbar any button.
I can do this if I rename temporarily the Chrome folder.

![20210507 080319  Firefox 89 0](https://user-images.githubusercontent.com/6840865/117407058-46f81b00-af0e-11eb-9da5-55b53c58d81d.jpg)

**Comment by nicolaasjan** [comment]
> However in Customize I can't drag from/to the Navigation Toolbar any button.
> I can do this if I rename temporarily the Chrome folder.

Here in 89.0b9 I can drag from/to the Navigation Toolbar just fine.
Maybe @Aris-t2 knows what can cause that behaviour.

**Comment by MrTatiTunes** [comment]
> 
> 
> > I guess, if I may ask, pack me all your Chrome folder and send me it zipped we will see if it works identically.
> 
> This is for Nightly (I think you only need `userChrome.css`):
> [userChrome.css.zip](https://github.com/Aris-t2/CustomCSSforFx/files/6400767/userChrome.css.zip)
> 
> I can't tell if it will work exactly the same for you though, because I'm on a Linux system.
> 
> > How you set the black motif? my is Light.
> 
> I just changed the theme to Dark.
> Menu Extra: ---> Addons and Themes ---> Themes ---> Dark
> 
> And I have set a new integer value in`about:config` called: `ui.systemUsesDarkTheme` with the value`1`
> Then internal Firefox pages will also be dark themed.
> Don't know if that works on Windows.
> 
> > You have as well a bit different spacing among ribbons.
> 
> That's maybe because I have set `browser.compactmode.show` to true in `about:config`, so the option becomes available in the "Customize" window:
> 
> ![screenshot-compact](https://user-images.githubusercontent.com/14093220/116603171-eb85c600-a92c-11eb-80af-425f8175e041.png)

Have no option compact...QQQ

**Comment by hopalongrock** [comment]
Using @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css
In 90.0a1 (2021-05-07) (64-bit) Customize overwrites on the upper part of the screen: the navigation toolbar and the bookmarks toolbar.

![20210508 092723  FC_W_ D__Xnap_DOWNLOAD](https://user-images.githubusercontent.com/6840865/117530969-6495b500-afe0-11eb-852e-8bbf6a10ba2d.jpg)

with Customize:

![20210508 092743  Customize Nightly — Firefox Nightly](https://user-images.githubusercontent.com/6840865/117530993-770fee80-afe0-11eb-8bc3-6aa68d3b972e.jpg)

**Comment by MrTatiTunes** [comment]
> 
> 
> Using @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css
> In 90.0a1 (2021-05-07) (64-bit) Customize overwrites on the upper part of the screen: the navigation toolbar and the bookmarks toolbar.
> 
> ![20210508 092723 FC_W_ D__Xnap_DOWNLOAD](https://user-images.githubusercontent.com/6840865/117530969-6495b500-afe0-11eb-852e-8bbf6a10ba2d.jpg)
> 
> with Customize:
> 
> ![20210508 092743 Customize Nightly — Firefox Nightly](https://user-images.githubusercontent.com/6840865/117530993-770fee80-afe0-11eb-8bc3-6aa68d3b972e.jpg)

I see that you have no arrows neither... QQ

**Comment by MrTatiTunes** [comment]
seems someone who did this code was doing it for MAC... somewhere in this motif there is lack of keyboard_arrow_left and keyboard_arrow_right for windows

**Comment by MrTatiTunes** [comment]
I do not like this way that GitHub is working... why I can not respond without quoting to direct member or mention him...

**Comment by nicolaasjan** [comment]
> I do not like this way that GitHub is working... why I can not respond without quoting to direct member or mention him...

You can actually. :)
Like this: @hopalongrock will work.

**Comment by hopalongrock** [comment]
@MrTatiTunes 

Unfortunately I don't understand what you say - here the left / right arrow in front of the Address Bar work as go back / forward one page.

On the the tab bar, when there are many tabs, is a < (less then) and a > (greater then) symbol to shift left/right.

Where should be this Keyboard arrow Left / Right?

@nicolaasjan 

Thank  you, please pack it, I'll try.

**Comment by nicolaasjan** [comment]
> Thank you, please pack it, I'll try.

It's in [this post](https://github.com/Aris-t2/CustomCSSforFx/issues/339#issuecomment-829506114) here. :)

**Comment by hopalongrock** [comment]
Sorry, no change with this userChrome.css, the same customize issue seems, 
(navigation toolbar and the bookmarks toolbar)

**Comment by nicolaasjan** [comment]
> Sorry, no change with this userChrome.css, the same customize issue seems,
> (navigation toolbar and the bookmarks toolbar)

I have no idea what could cause this behaviour...Sorry.

**Comment by hopalongrock** [comment]
Trying with custom_css_for_fx_v3.2.0pre3.zip in FX89 and 90

There is no change using
@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
Using
@import "./css/tabs/tabs_below_navigation_toolbar_fx89.css"; /**/
there is no customize problem, but in the tab bar the selected tab is higher as the unselected ones, and does not fit in the tab bar

![20210509 125131  Beta 89 0 Mozilla Firefox](https://user-images.githubusercontent.com/6840865/117569304-638a8380-b0c5-11eb-91c9-137e0a53f9b0.jpg)

**Comment by nicolaasjan** [comment]
Regarding writing : 
> @import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/**

You've actually inserted a link to:
https://github.com/import ;)

It's better to put code snippets between backticks:
`@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/`

**Comment by nicolaasjan** [comment]
> Thank you, I don't understand the // at the end of line,

That's because GitHub displays your `/**/**` as /**/** (second slash is bold).

**Comment by MrTatiTunes** [comment]
> 
> 
> @MrTatiTunes
> 
> Unfortunately I don't understand what you say - here the left / right arrow in front of the Address Bar work as go back / forward one page.
> 
> On the the tab bar, when there are many tabs, is a < (less then) and a > (greater then) symbol to shift left/right.
> 
> Where should be this Keyboard arrow Left / Right?
> 
> @nicolaasjan
> 
> Thank you, please pack it, I'll try.

https://user-images.githubusercontent.com/30302908/117563339-227e7900-b09d-11eb-9f97-91a957be219b.png
I do not understand what you do not understand :)

**Comment by hopalongrock** [comment]
@MrTatiTunes 

I'm sorry, I missed your original picture, now I see you talk about the  <  and > icon on the right and left edge of the tab bar,  I didn't know that these are called arrow.
I am on Windows 10,, FX89 Beta, FX90 Nightly and use now
@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css";
previously
@import "./css/tabs/tabs_below_navigation_toolbar_fx89.css";
in userChrome.css,
and both show the < and > icons and scroll fine

**Comment by MrTatiTunes** [comment]
@hopalongrock 
Yes and code in HTML is keyboard_arrow_left ETC.
I am on W7, but it should not matter ... 
Did not test it on 10 and Enterprise LTSC yet… too busy with RL... and W7 rules ;) still the best system, even better without F*** MS interfering in it.... (but they still try to)

**Comment by MrTatiTunes** [comment]
> 
> 
> > I do not like this way that GitHub is working... why I can not respond without quoting to direct member or mention him...
> 
> You can actually. :)
> Like this: @hopalongrock will work:
> 
> > @hopalongrock, if you need, I can pack you my chrome...
> > and you will have it working

Thanks … did not see it before!!! :)

**Comment by MrTatiTunes** [comment]
@nicolaasjan 
this is strange, there is different interpreter in Nightly for 7 and w10… on W10 arrows are present...
![image](https://user-images.githubusercontent.com/30302908/117733501-9168e880-b1e9-11eb-90b8-2ce6d1eca8fb.png)

**Comment by MrTatiTunes** [comment]
I am still using my W7 and I find it the best system so far second is LTSC, if gamer on Windows platform it does not allow playing Microsoft games the same way as 7. So 10 I use to test Microsoft Store and other this kind of Corpo inventions... loll

**Comment by hopalongrock** [comment]
In Customize "Restore Defaults" solved my Customize screen issue (see https://github.com/Aris-t2/CustomCSSforFx/issues/339#issuecomment-835175589)

I made this in FF 88, 89, 90 versions, and Customize now works fine in all three versions.

**Comment by nicolaasjan** [comment]
@Aris-t2 
In Nightly 2021-05-14 the menu bar is now invisible...

![screenshot-invisible-menubar](https://user-images.githubusercontent.com/14093220/118352089-d4b3b600-b55f-11eb-8e4a-38b1cc0c902d.png)

**Comment by hopalongrock** [comment]
No change here, menu bar is visible.
Using 
`@import "./css/tabs/tabs_below_navigation_toolbar_fx65_v2.css"; /**/
`

![20210515 111554  About Firefox Nightly](https://user-images.githubusercontent.com/6840865/118355204-1bf57300-b56f-11eb-8b33-25672a772167.jpg)

**Comment by nicolaasjan** [comment]
@hopalongrock 
The problem is with the Dark Theme...
Light theme is OK, but hurts my eyes:

![screenshot-light-theme](https://user-images.githubusercontent.com/14093220/118355831-8f4cb400-b572-11eb-8eb7-f398c6051d55.png)

**Comment by hopalongrock** [comment]
It looks fine with the dark theme too.

![20210515 120704  MajorGeeks Com - MajorGeeks — Firefox Nightly](https://user-images.githubusercontent.com/6840865/118356600-50206200-b576-11eb-9fae-d8aaf6c9a0f6.jpg)

**Comment by hopalongrock** [comment]
@nicolaasjan 
Are you sure that the Menu Bar is enabled in customize?

**Comment by nicolaasjan** [comment]
> @nicolaasjan
> Are you sure that the Menu Bar is enabled in customize?

Yes.
I can even click on the invisible items in it.
It might be OS specific (I'm on Linux).

![screenshot-press](https://user-images.githubusercontent.com/14093220/118357034-43047280-b578-11eb-94ce-17cdfb1d9a56.png)

**Comment by hopalongrock** [comment]
I have only Windows.

A bit similar was that in Customize the navigator toolbar and the bookmark toolbar were not visible or only partially visible.
But if I renamed UserChrone.css, it worked normally.

Then Customize - Restore Defaults helped me - naturally I had to enable the toolbars, and move the buttons to their place.

**Comment by nicolaasjan** [comment]
Strange.
I just tried again in a Virtual Machine (Debian Linux) and there is no such problem...

**Comment by MrTatiTunes** [comment]
@nicolaasjan @hopalongrock 
RECENT UPDATE THE ARROWS ARE THERE BUT THEY ARE WHITE ON WHITE BACKGROUND OR BLACK ON BLACK... THIS IS JUST THE ERROR IN ASSIGNING COLORS TO THEM.
THEY ARE HARDLY VISIBLE.
![obraz](https://user-images.githubusercontent.com/30302908/118372839-68f73a80-b5ab-11eb-9687-3d26e3ddc14d.png)

**Comment by MrTatiTunes** [comment]
@nicolaasjan @hopalongrock 
Hi guys again, I guess there is improvement in Nightly, and finally it works better.
But the previous behavior of tabs (before Nightly 90) was a bit nicer as the actually visible tab in tabs bar was nicely accented, 
in this theme both light and dark there is no significant/visible difference to other tabs.
![obraz](https://user-images.githubusercontent.com/30302908/118385229-2c582d00-b605-11eb-992b-7ce48831bca3.png)

the previous userConfig.css is still working for me in regular Firefox and EMEA
![obraz](https://user-images.githubusercontent.com/30302908/118385290-a38dc100-b605-11eb-87c0-df868201de83.png)

in this theme it doesn't...
Have you any solution for this improvement?

**Comment by MrTatiTunes** [comment]
In default system motif it looks like this:
![obraz](https://user-images.githubusercontent.com/30302908/118385369-257dea00-b606-11eb-8464-bd46dc169207.png)

**Comment by Speravir** [comment]
In update for [my comment on 24 April](#issuecomment-826012957):

It is now planned to remove the setting for disabling Proton with version 91 of Firefox, cf. [Moz Bugzilla ticket 1709425](https://bugzilla.mozilla.org/show_bug.cgi?id=1709425).

**Comment by Aris-t2** [comment]
Thanks for the note.

**Comment by glennpc** [comment]
Sadly, browser.proton.contextmenus.enabled has just been removed from Nightly 91 as of today.

I'm going to stay with FF 89 or FF 90 if that's the case when it gets released in August.

**Comment by nicolaasjan** [comment]
> Sadly, browser.proton.contextmenus.enabled has just been removed from Nightly 91 as of today.
> 
> I'm going to stay with FF 89 or FF 90 if that's the case when it gets released in August.

These as well:
`browser.proton.appmenu.enabled`
`browser.proton.tabs.enabled`

My Nightly (2021-06-04) looks like this now (tabs wrong):
![Screenshot-Nightly-20210605](https://user-images.githubusercontent.com/14093220/120882443-55eaef80-c5d8-11eb-9e4a-3bf8e2e07fa4.png)

**Comment by MrTatiTunes** [comment]
> 
> 
> > Sadly, browser.proton.contextmenus.enabled has just been removed from Nightly 91 as of today.
> > I'm going to stay with FF 89 or FF 90 if that's the case when it gets released in August.
> 
> These as well:
> `browser.proton.appmenu.enabled`
> `browser.proton.tabs.enabled`
> 
> My Nightly (2021-06-04) looks like this now (tabs wrong):
Hmm, what is wrong there...?

**Comment by Aris-t2** [comment]
You now have to set `browser.proton.enabled` to `true` on Nightly or most code changes made since v3.2.0 won't be applied.

**Comment by nicolaasjan** [comment]
> You now have to set `browser.proton.enabled` to `true` on Nightly or most code changes made since v3.2.0 won't be applied.

Thanks, did that and it's better now.
One strange thing though with the context menu (spacing and alignment):
![Schermafdruk op 2021-06-05 10-17-36](https://user-images.githubusercontent.com/14093220/120885120-93a34480-c5e7-11eb-980e-3d610b55c735.png)

**Comment by Corben78** [comment]
As I was struggling with having the tab bar below the nav bar as well, I was even thinking about keeping it now as it is and having it above the nav bar. But a [friendly person on mastodon](https://linuxrocks.online/@friend/106356955689221548) gave me this snippet and the tab bar is now beautifully below the nav bar:
```css
#navigator-toolbox {
  display: flex;
  flex-direction: column;
}

#titlebar {
  order: 5;
}
```
Maybe this can be worked into CustomCSSforFx? Atm I have this in my custom userChrome.css file.

**Comment by Aris-t2** [comment]
Unfortunately this only works an a very specific ui setup:
- OS titlebar (issues with Fx titlebar)
- menubar disabled (menubar looks misplaced between bookmarks toolbar and tabs toolbar)


But it is of course a great solution for that specific setup.

**Comment by fayer3** [comment]
is there a way to fix shrinking tabs when dragging with the classic tabs and proton or do I need to disable proton?

https://user-images.githubusercontent.com/8282299/120894907-024dc580-c61b-11eb-9319-373aae180d5f.mp4

**Comment by mzso** [comment]
[**@Aris-t2**](https://github.com/Aris-t2) commented on [2021. ápr. 10. 18:45 CEST](https://github.com/Aris-t2/CustomCSSforFx/issues/339#issue-855108522 "2021-04-10T16:45:14Z - Replied by Github Reply Comments"):
> 
> Other Proton settings:  
> `browser.proton.infobars.enabled > false`  
I don't have this in 88. Does it main they only changed infobars in 89?

> `browser.proton.modals.enabled > false`  

What does this mean in this context?

> `browser.proton.places-tooltip.enabled > false`  

I can't see this affecting anything. Looked at history sidebar and in page tooltips. What's this setting's supposed to effect?

**Comment by Aris-t2** [comment]
@fayer3 
This should be fixed on 3.2.2

@mzso 
You can ignore non-available settings. There were present by the time a wrote the op and some of them most likely were temporarily added as placeholders for testing purposes.

**Comment by Aris-t2** [comment]
This thread is not needed anymore.

Continue here https://github.com/Aris-t2/CustomCSSforFx/issues/368

**Comment by mzso** [comment]
[**@Aris-t2**](https://github.com/Aris-t2) commented on [2021. jún. 6. 14:57 CEST](https://github.com/Aris-t2/CustomCSSforFx/issues/339#issuecomment-855395156 "2021-06-06T12:57:08Z - Replied by Github Reply Comments"):
> [@fayer3](https://github.com/fayer3)  
> This should be fixed on 3.2.2
> 
> [@mzso](https://github.com/mzso)  
> You can ignore non-available settings. There were present by the time a wrote the op and some of them most likely were temporarily added as placeholders for testing purposes.

Great. Though the last two are available, I just don't know what they're supposed to effect to check.
